screen2.0/src/app/downloads/datamatrices.tsx

```diff
@@ -237,24 +290,24 @@ export function DataMatrices() {
   const selectorButton = (variant: Selected) => {
```

Comment:
Can you center the text in these buttons? Select H3K4me3 to see what I mean. BorderLeft is bumping text over to the right